native/cocos/renderer/pipeline/custom/RenderCommonTypes.h

```diff
@@ -104,6 +104,8 @@ enum class ResourceFlags : uint32_t {
     SHADING_RATE = 0x80,
     TRANSFER_SRC = 0x100,
     TRANSFER_DST = 0x200,
+    VERTEX = 0x400,
+    INDEX = 0x800,
```

Comment:
I believe we don't support vertex/index state changes in RenderGraph/FrameGraphDispatcher? @hana-alice

Reply:
@stanleyljl said we will have gpu generated index buffer. So these states are needed.

Reply:
Forgot to submit review, OK will preserve